src/components/molecules/plan/plan.html

```diff
@@ -0,0 +1,29 @@
+<div class="wrap">
+  <div class="plan">
+    <h2 class="plan__title">プランテキスト</h2>
+    <div class="plan__price-outer">
+      <p class="plan__price"><span class="and">¥</span>5,000</p>
+      <p class="plan__month__price">月額</p>
+    </div>
+    <ul class="plan__item">
+      <li class="plan__item-list">入会金0円</li>
+      <li class="plan__item-list">平日 : 9時〜17時</li>
+      <li class="plan__item-list">土日祝 : 利用ナシ</li>
+      <li class="plan__item-list">コアタイムでの利用に最適なプラン</li>
+    </ul>
+  </div>
+
+  <div class="plan plan--std">
```

Comment:
細かいですが、「スタンダードプラン」という名称は今後変わる可能性があり、そうした場合クラス名と実態がずれてしまいます。
なるべくコンテンツの内容に依存したクラス名を避けた方が、この先の変更に強くなると思います。
```suggestion
  <div class="plan plan--bold">
```
パッと思いつくのはboldとかですが、もっといいのがあるかもしれません。